source/documentation/06-api-reference.md

```diff
@@ -80,6 +80,7 @@ These are the known status codes you are likely to receive:
 |404 | The resource you want cannot be found|
 |412 | Precondition failed: e.g. mismatch in expected refund amount available|
 |422 | Unprocessable entity obtained on a request validation|
+|429 | Too many requests. Request rate is above the rate limit allowed|
```

Comment:
Perhaps there isn't a need for the word "allowed" here so we could just say:

> |429 | Too many requests. Request rate is above the rate limit|

Reply:
Agree. Removed word 'allowed'